### ISSUE TITLE
Document accessibility resources

### ISSUE THREAD

**Issue by NicMcPhee** [UMM-CSci-3601/3601-iteration-template#45]
Various people shared a variety of resources about accessibility (in general and specific to Angular) in the S20 Slack, and we should pull those together into a MD page here in this repo for next year.

**Comment by NicMcPhee** [comment]
Some things from Slack:

- "Firefox has accessibility tools" (other browsers may as well)
- The NoCoffee browser extension allows you to simulate a variety of visual impairments.
- https://angular.io/guide/accessibility
- https://accessibility.umn.edu/core-skills-web-developers

**Comment by floogulinc** [comment]
More tools:
- Audit your Angular app's accessibility with codelyzer https://web.dev/accessible-angular-with-codelyzer/
- WAVE Web Accessibility Evaluation Tool https://wave.webaim.org/

**Comment by NicMcPhee** [comment]
Should we create a simple `ACCESSIBILITY.md` with a short intro paragraph and a bullet list of resources? It wouldn't be a lot, but it would be easy and better than nothing?

**Comment by NicMcPhee** [comment]
@floogulinc suggested on Slack that we just add these to the resources section of the REAMDE, so I'll do that.

**Comment by NicMcPhee** [comment]
Should be closed by ac11734dc441709e52a2ca5fb23a96571cea0b60 if/when that is merged in.
